src/Discord.Net.Core/Entities/Activities/GameParty.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Discord
+{
+    public class GameParty
+    {
+        internal GameParty() { }
+
+        public string Id { get; internal set; }
+        public int[] Size { get; internal set; }
```

Comment:
Array of Int? This probably has a better way of being exposed. Assuming it's two ints, count and cap, a maybe a ValueTuple?